unity/README.md

```diff
@@ -4,6 +4,13 @@ This is Unity's fork of the .NET Runtime repository.
 
 The difference between this fork and the upstream repository should be as small as possible, with all of the differences specific to Unity. Our goal is to upstream as many changes made here as possible.
 
+
+## Environment requirements
+
+Please refer to this page for environment requirements before build:
+https://github.com/dotnet/runtime/tree/main/docs/workflow/requirements
```

Comment:
Could you make this a mark down link so it's clickable?

I think the following should work:

Please refer to [this page](https://github.com/dotnet/runtime/tree/main/docs/workflow/requirements) for environment requirements before building.